common/pom.xml

```diff
@@ -37,6 +67,10 @@
       <scope>compile</scope> <!-- override the 'test' scope defined at parent pom.xml -->
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jctools</groupId>
+      <artifactId>jctools-core</artifactId>
+    </dependency>
```

Comment:
I tested on my current project and netty-common nor netty-all are pulling jctools as a dependency.

I also looked at the deployed pom and it only has the shade section but the dependency is not there so I'm guessing that:

Yes, it is using the reduced pom.